### PR TITLE
LPS-99308 Apply Liferay Plugin to hybrid projects

### PR DESCRIPTION
**Pull request by christopherbryanboyd**
Hi @gamerson , here is the fix for https://issues.liferay.com/browse/LPS-99308

### CODE REVIEW

File: modules/sdk/gradle-plugins-workspace/src/gradleTest/angularTest/build.gradle [hunk 1]
```diff
@@ -0,0 +1,5 @@
+task runGradleTest
+
+runGradleTest {
+	dependsOn ":clean"
```
Comment (gamerson) [inline modules/sdk/gradle-plugins-workspace/src/gradleTest/angularTest/build.gradle:4]: Can you update this test to have 3 types of modules
1) osgi portlet
2) hybrid java/js portlet (like this one has)
3) pure js-portlet

Then assert that 1 has LiferayOSGiplugin applied, 2 has LiferayOSGi applied, and 3) has FrontendPlugin applied

Comment (christopherbryanboyd) [reply modules/sdk/gradle-plugins-workspace/src/gradleTest/angularTest/build.gradle:4]: Yes, I will work on that now.

**Comment by gamerson** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by gamerson** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by gamerson** [comment]
sent here: https://github.com/petershin/liferay-portal/pull/611

**Pull request closed without merging**